eslint-plugin/tests/lib/rules/image-src-attribute-not-empty.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2023 Green Code Initiative
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/image-src-attribute-not-empty");
+const RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2021,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+});
+const expectedError1 = {
+  messageId: "SpecifySrcAttribute",
+  type: "JSXAttribute",
+};
+const expectedError2 = {
+  messageId: "SpecifySrcAttribute",
+  type: "JSXOpeningElement",
+};
+
+ruleTester.run("image-src-attribute-not-empty", rule, {
+  valid: [
+    `
+      <img src='logo.svg' alt='Ceci est une image svg'/>
```

Comment:
Can you translate all alternative attributes in this file?
```suggestion
      <img src='logo.svg' alt='This is a SVG image'/>
```